lib/aur-fetch

```diff
@@ -33,6 +33,7 @@ while true; do
         -L|--write-log) shift; log_dir=$1 ;;
         -r|--recurse)   recurse=1 ;;
         -v|--verbose)   verbose=1 ;;
+        --force)        fetch_args+=('--force') ;;
```

Comment:
Almost forgot: Maybe add an `-f` short option as well?

Reply:
That's what the pacman devs once thought ;)